4.classify-single-cell-phenotypes/4a.classify-single-cell-phenotypes/README.md

```diff
@@ -0,0 +1,56 @@
+# 4a. Classify Single Cell Phenotypes
+
+In this module, we present our pipeline for classifying single-cell phenotypes.
+
+## Deriving Classifications
+
+Follow the steps in [4.classify-single-cell-phenotypes/README.md](../README.md) to run this module.
+
+## File Description
+
+This module outputs compressed csv files containing the classification probabilities of preprocessed Cell-Health features output by [3.preprocess-features](../../3.preprocess-features/).
+These classifications are derived per plate by each of the models created in the [phenotypic_profiling](https://github.com/WayScience/phenotypic_profiling_model) repository.
+These models include multi-class models (16 output probabilities, 1 for each phenotypic class) and single-class models (2 output probabilities, positive and negative probabilities for the model's particular phenotypic class).
+For more information on model training and type see [phenotypic_profiling_model/2.train_model](https://github.com/WayScience/phenotypic_profiling_model/tree/main/2.train_model).
+
+Each compressed csv file generated by this module includes rows of single-cell metadata (location, perturbation, etc) along with feature classification probabilities, which is determined by the specific model used.
+There is one compressed csv file corresponding to each of the nine plates in the Cell-Health dataset.
+The contents of the compressed csv file containing `OutOfFocus` model predictions from plate `SQ00014613` are shown below:
+
+| Location_Center_X | Location_Center_Y | Metadata_Site | Metadata_Well | Metadata_Plate | Metadata_Plate_Map_Name | Metadata_Reagent | OutOfFocus | OutOfFocus Negative |
```

Comment:
Great documentation! Consider just including the table in a notebook, or reducing the number of example entries.